src/lib/prompts/organic-cta.ts

```
@@ -0,0 +1,398 @@
+  'Follow for part 2',
+]
+
+/**
+ * CTA selection guide based on hook type and content
+ */
+export const CTA_SELECTION_GUIDE: Record<string, OrganicCTAType[]> = {
+  // Hook types -> recommended CTAs
+  bold_statement: ['qualifier_challenge', 'rhetorical_close', 'exclusivity_signal'],
+  question: ['rhetorical_close', 'fantasy_invitation', 'emotional_bond'],
+  confession: ['emotional_bond', 'consequence_lock', 'rhetorical_close'],
+  challenge: ['qualifier_challenge', 'direct_desire', 'exclusivity_signal'],
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)